### PR TITLE
Data release 20240507

### PR DESCRIPTION
**Pull request by hareesh-ms**

### CODE REVIEW

File: internal/server/stat/golden/get_stat_all/result.json [hunk 1]
```diff
@@ -671,15 +671,6 @@
               "provenanceDomain": "census.gov",
               "provenanceUrl": "https://www.census.gov/programs-surveys/acs/data/data-via-ftp.html"
             },
-            {
-              "val": {
-                "2011": 311591917
-              },
-              "measurementMethod": "HumanCuratedStats",
-              "importName": "HumanCuratedStats",
```
Comment (hareesh-ms) [inline internal/server/stat/golden/get_stat_all/result.json:679]: Moved to searchonly import group

File: 1 [hunk 2]
```diff
@@ -0,0 +1,1464 @@
+{
```
Comment (chejennifer) [inline 1:1]: what is this file? why is it just named 1?

File: internal/server/v0/placestatvar/golden/get_place_stat_vars/california.json [hunk 3]
```diff
@@ -8022,46 +8022,6 @@
         "Count_Person_18To64Years_PovertyStatusDetermined",
         "Count_Person_18To64Years_PovertyStatusDetermined_ResidesInGroupQuarters",
         "Count_Person_18To64Years_PovertyStatusDetermined_ResidesInNoninstitutionalizedGroupQuarters",
-        "Count_Person_18To64Years_RatioToPovertyLine1.38To4",
-        "Count_Person_18To64Years_RatioToPovertyLine1.38To4_BlackOrAfricanAmericanAlone",
-        "Count_Person_18To64Years_RatioToPovertyLine1.38To4_Female",
```
Comment (chejennifer) [inline internal/server/v0/placestatvar/golden/get_place_stat_vars/california.json:8027]: curious why these all got deleted?

File: internal/server/v1/propertyvalues/golden/bulk_property_values_in/typeOf.json [hunk 4]
```diff
@@ -9683,236 +9751,6 @@
           "dcid": "wikidataId/Q1005905",
           "provenanceId": "dc/base/WikidataOtherIdGeos"
         },
-        {
-          "name": "Borek Wielkopolski",
-          "types": [
```
Comment (chejennifer) [inline internal/server/v1/propertyvalues/golden/bulk_property_values_in/typeOf.json:9688]: are these deletions expected?

File: internal/server/v2/observation/golden/variable/result.json [hunk 5]
```diff
@@ -6352,11 +6352,6 @@
         "wikidataId/Q506877": {}
       }
     },
-    "minimumWage": {
```
Comment (chejennifer) [inline internal/server/v2/observation/golden/variable/result.json:6355]: is this deletion expected?

**Review by chejennifer — COMMENTED** [review]
thanks! Just curious if some deletions are expected

File: 1 (hunk 2 above)
Comment (hareesh-ms) [reply 1]: This seems to be added by mistake. Removed.

File: internal/server/v0/placestatvar/golden/get_place_stat_vars/california.json (hunk 3 above)
Comment (hareesh-ms) [reply internal/server/v0/placestatvar/golden/get_place_stat_vars/california.json]: The deletions are removed after the SAHIE import update.

File: internal/server/v1/info/golden/variable_group_info/demographics.json [hunk 6]
```diff
@@ -117,20 +108,14 @@
...
+        "descendentStatVarCount": 27225
       },
       {
         "id": "dc/g/Person_AgeGroupClassification",
         "specializedEntity": "Age Group Classification",
         "displayName": "Person by Age Group Classification",
         "descendentStatVarCount": 2
       },
-      {
-        "id": "dc/g/Household_BenefitsStatus",
```
Comment (ajaits) [inline internal/server/v1/info/golden/variable_group_info/demographics.json:129]: I assume the dc/g/Household* deletions are due to the vertical_spec changes for hourshold.

File: internal/server/v2/observation/golden/variable/result.json (hunk 5 above)
Comment (hareesh-ms) [reply internal/server/v2/observation/golden/variable/result.json:6355]: wikidataId/Q1951 , minimumWage combination is missing in Production as well.

**Review by ajaits — APPROVED** [review]

**Review by juliawu — APPROVED** [review]

**Pull request merged**